system/core/HyphaSession.php

```diff
@@ -29,7 +29,6 @@ public function __construct(HyphaRequest $request) {
 			// accessible to scripts).
 			session_name('hyphaSession');
 			ini_set('session.cookie_path', $request->getRootUrlPath());
-			ini_set('session.cookie_secure', $request->isSecure());
 			ini_set('session.cookie_httponly', true);
```

Comment:
Why not put the set secure within an if statement?

Reply:
How do you mean exactly? Only set the property when `isSecure()` is true? That is what the old code effectively did, but that breaks mixed HTTP/HTTPS setups. Or am I misunderstanding you?

Writing this, I do realize that maybe we could include http/https in the session name so really support two different sessions for http and https *and* have secure cookies?